inventory-consumer/handler.go

```diff
@@ -1,15 +1,52 @@
 package inventoryconsumer
 
 import (
+	"config-manager/domain"
+	"config-manager/domain/message"
+	kafkaUtils "config-manager/infrastructure/kafka"
 	"context"
+	"encoding/json"
 	"fmt"
 
 	kafka "github.com/segmentio/kafka-go"
 )
 
 type handler struct {
+	ConfigManagerService domain.ConfigManagerInterface
 }
 
 func (this *handler) onMessage(ctx context.Context, msg kafka.Message) {
-	fmt.Println("Message: ", msg.Value)
+	eventType, err := kafkaUtils.GetHeader(msg, "event_type")
+	if err != nil {
+		fmt.Println("Error getting header: ", err)
+		return
+	}
+
+	if eventType == "delete" {
```

Comment:
\<nitpick\/\> It may be better to ignore events where type is not one of known supported ones (created, updated) rather than on the unsupported one (delete). It does not make a difference currently but would make the code more stable should inventory introduce a new event type (which is not likely, but possible).

Reply:
Good call!